shared/common-adapters/clickable-box.js.flow

```diff
@@ -2,10 +2,12 @@
 import React, {Component} from 'react'
 
 export type Props = {
+  children?: any,
   style?: Object,
-  onClick: () => void,
-  hoverColor?: ?string,
-  backgroundColor?: ?string,
+  onClick?: ?() => void,
```

Comment:
I don't think this should be an optional key. It's a little tricky. See:

https://github.com/keybase/client/pull/4116/files#r76900386